packages/server/package.json

```diff
@@ -48,7 +48,7 @@
     "lodash-es": "^4.17.15",
     "merge-stream": "2.0.0",
     "nanoid": "3.1.10",
-    "next": "9.5.3",
+    "next": "^9.5.5",
```

Comment:
This needs to be an exact version number.

```suggestion
    "next": "9.5.5",
```

Reply:
Okay just did